client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -44,10 +44,10 @@ class DisconnectSite extends Component {
 					<FormattedHeader
 						headerText={ translate( 'Disconnect Site' ) }
 						subHeaderText={ translate(
-							'Tell us why you want to disconnect your site from WordPress.com.'
+							"We'd love to know why you're disconnecting -- it will help us improve Jetpack."
```

Comment:
I don't think we want a literal two dashes `--` here. I think we want [em dash](http://www.thepunctuationguide.com/em-dash.html), `—`, perhaps @keoshi can confirm?